qterminal/qterminal_fr.ts

```diff
@@ -0,0 +1,703 @@
+        <translation>&amp;Nouvelle fenêtre</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="373"/>
+        <source>&amp;Hide Window Borders</source>
+        <translation>Cac&amp;her les bordures</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="388"/>
+        <source>&amp;Show Tab Bar</source>
+        <translation>&amp;Afficher la barre des onglets</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="400"/>
+        <source>Fullscreen</source>
+        <translation>Plain écran</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="420"/>
+        <source>&amp;Bottom</source>
+        <translation>&amp;Bas</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="421"/>
+        <source>&amp;Top</source>
+        <translation>&amp;Haut</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="422"/>
+        <location filename="../../src/mainwindow.cpp" line="453"/>
+        <source>&amp;Right</source>
+        <translation>&amp;Droite</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="423"/>
+        <location filename="../../src/mainwindow.cpp" line="454"/>
+        <source>&amp;Left</source>
+        <translation>&amp;Gauche</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="438"/>
+        <source>&amp;Tabs Layout</source>
+        <translation>&amp;Disposition des onglets</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="452"/>
+        <source>&amp;None</source>
+        <translation>&amp;Aucun</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="470"/>
+        <source>S&amp;crollbar Layout</source>
+        <translation>Disposition de la barre de &amp;défilement</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="481"/>
+        <source>&amp;BlockCursor</source>
+        <translation>&amp;Bloquer le curseur</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="482"/>
+        <source>&amp;UnderlineCursor</source>
+        <translation>&amp;Souligner le curseur</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="483"/>
+        <source>&amp;IBeamCursor</source>
+        <translation>Curseur en forme de &amp;I</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.cpp" line="499"/>
+        <source>&amp;Keyboard Cursor Shape</source>
+        <translation>Curseur en forme de &amp;clavier</translation>
```

Comment:
No. This is not a cursor looking like a keyboard but how the cursor should look like when the keyboard is used.
You may just tweak the setting at Preferences - Appearance - Keyboard cursor Shape to see the meaning.